sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py

```diff
@@ -613,7 +615,7 @@ def run_named_trajectory_unsafe(self, trajectory, wait=False):
                             - pause_time -> time to wait at this wp
         """
         joint_trajectory = self.make_named_trajectory(trajectory)
-        if joint_trajectory is not None:
+        if type(joint_trajectory) == JointTrajectory:
```

Comment:
we should add some info whan its not that.
so
if type(joint_trajectory) != JointTrajectory:
    bla bla
   return
self.run_joint_trajectory_unsafe(joint_trajectory, wait)